sdk/storage/azure-storage-file-share/CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
 # Release History
 
-## 12.21.0b1 (Unreleased)
+## 12.21.0b1 (2024-12-10)
 
 ### Features Added
+- Added support for service version 2025-05-05.
+- Added support for `ShareFileClient` using the `NFS` protocol to `create_hard_link` 
+with an existing file by specifying full path starting from the root.
+- Added support for `ShareFileClient` using the `NFS` protocol to copy an existing file through the 
+`start_copy_from_url` API by specifying optional keywords `owner`, `group`, `file_mode`.
```

Comment:
nit: "`group`, **and** `file_mode`.

Reply:
Let's refactor this bullet to just more generally mention `owner`, `group` and `file_mode` across all APIs. Something like:
"Added support for NFS shares to existing APIs. This includes adding `owner`, `group`, and `file_mode` options to LIST APIS HERE and returning `owner`, `group`, and `file_mode` in `DirectoryProperties` and `FileProperties`.